src/camlsnark_c/libsnark-caml/doxygen.conf

```diff
@@ -40,7 +40,7 @@ PROJECT_NUMBER         =
 
 PROJECT_BRIEF          =
```

Comment:
this file as well, could you open a PR on their repo, and revert the changes here?

Reply:
https://github.com/scipr-lab/libsnark/pull/128